src/coreclr/jit/compiler.cpp

```diff
@@ -1262,8 +1262,12 @@ void DisplayNowayAssertMap()
             fout = _wfopen(strJitMeasureNowayAssertFile, W("a"));
             if (fout == nullptr)
             {
+#if !defined(TARGET_WINDOWS)
```

Comment:
```suggestion
#if !defined(HOST_WINDOWS)
```
Seems there is not an easy way to do this.